dev_apps/music-nga/index.html

```diff
@@ -31,31 +40,51 @@ <h1 id="header-title">Music</h1>
   </music-view-stack>
 
   <music-tab-bar id="tab-bar">
-    <button value="home" data-url="/home" data-icon="grid"></button>
-    <button value="playlists" data-url="/playlists" data-icon="playlist"></button>
-    <button value="artists" data-url="/artists" data-icon="artist"></button>
-    <button value="albums" data-url="/albums" data-icon="album"></button>
-    <button value="songs" data-url="/songs" data-icon="songs"></button>
+    <button value="home"
+        data-url="/home"
+        data-icon="grid"
+        data-l10n-id="home-tab">
+    </button>
+    <button value="playlists"
+        data-url="/playlists"
+        data-icon="playlist"
+        data-l10n-id="playlists-tab">
+    </button>
+    <button value="artists"
+        data-url="/artists"
+        data-icon="artist"
+        data-l10n-id="artists-tab">
+    </button>
+    <button value="albums"
+        data-url="/albums"
+        data-icon="album"
+        data-l10n-id="albums-tab">
+    </button>
+    <button value="songs"
+        data-url="/songs"
+        data-icon="songs"
+        data-l10n-id="songs-tab">
+    </button>
   </music-tab-bar>
 
   <music-overlay id="empty-overlay"
-      heading="Add songs to get started"
-      message="Load songs on to the memory card." hidden>
+      heading-l10n-id="empty-title"
+      message-l10n-id="empty-text" hidden>
```

Comment:
Would it be possible to use projected content instead of `heading-l10n-id` and `message-l10n-id`?  That way you wouldn’t need to manually handle re-localization in `attributeChangedCallback`.